config.json

```diff
@@ -1289,6 +1289,15 @@
         "searching"
       ]
     },
+    {
+      "slug": "rest-api",
+      "uuid": "4c89b6a8-0e89-46c9-5a7e-31bfe6a57007",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 0,
+      "topics": null,
```

Comment:
Do you have any suggestions for topics? There is a reference list to be found [here](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt). From that list, I think not much is relevant here, but we might use a new topic here: "json". What do you think?

Reply:
json seems appropriate for this exercise. I would probably use parsing as well from the reference list.